ga4gh/backend.py

```diff
@@ -574,7 +569,8 @@ def variantAnnotationsGenerator(self, request):
         compoundId = datamodel.VariantAnnotationSetCompoundId.parse(
             request.variantAnnotationSetId)
         dataset = self.getDataRepository().getDataset(compoundId.datasetId)
-        variantAnnotationSet = dataset.getVariantAnnotationSet(
+        variantSet = dataset.getVariantSet(compoundId.variantSetId)
+        variantAnnotationSet = variantSet.getVariantAnnotationSet(
```

Comment:
From discussion with @diekhans and @david4096, this seems to go against a common use case in which a variant annotation set would come from a different dataset than the variant set.